src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -128,6 +136,29 @@ def close(self):
         # Constructor exceptions may occur prior to _session being set.
         if hasattr(self, '_session'):
             self._session.close()
+        if hasattr(self, '_https_session'):
+            self._https_session.clear()
+
+    @staticmethod
+    def _make_https_session(cert_location, key_location, ca_certs_path, auth, key_password):
+        https_session = urllib3.PoolManager(cert_reqs='CERT_REQUIRED', ca_certs=ca_certs_path,
+                                             cert_file=cert_location, key_file=key_location, key_password=key_password)
+        https_session.auth = auth
+        return https_session
+
+    def _send_https_session_request(self, url, method, headers, body):
+        request_headers = {'Accept': ACCEPT_HDR}
+        auth = self._https_session.auth
+        if body:
+            body = json.dumps(body).encode('UTF-8')
+            request_headers["Content-Length"] = str(len(body))
+            request_headers["Content-Type"] = "application/vnd.schemaregistry.v1+json"
```

Comment:
This should be application/json as body is being converted to Json in the above call. Please check.

Reply:
Actually it has to do with the interal working of the schema registry and was this for all the request with the body for the old flow as well!